src/components/Personalization/createPersonalizationDetails.js

```diff
@@ -10,7 +10,8 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { includes } from "../../utils";
+import { includes, isNonEmptyString, isNonEmptyArray } from "../../utils";
```

Comment:
Any reason we can't have `buildPageSurface, normalizeSurfaces` directly in `./utils`? You can still have a separate file, but it is easier to control everything through a top level file/module.

Reply:
@ninaceban Yes, in a previous comment @jonsnyder requested that we have all surface-related utils encapsulated in a separate module.